src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -228,6 +228,8 @@ case class WorkspacePermissionsPair(workspaceId: String,
 
 case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, String])
 
+case class BucketUsageResponse(usageInBytes: BigInt)
```

Comment:
@MatthewBemis do you think there would be objection to adding `final` to all of these? It's either that or leave it as is. I'm not going to break the established practice by making only this one final.